server/matchmaker/search.py

```diff
@@ -271,9 +271,8 @@ def raw_ratings(self):
         return list(itertools.chain(*[s.raw_ratings for s in self.searches]))
 
     @property
-    def failed_matching_attempts(self) -> List[int]:
-        """Used for logging so returning a different type here is fine"""
-        return [search.failed_matching_attempts for search in self.searches]
+    def failed_matching_attempts(self) -> int:
```

Comment:
may `nbr_of_failed_matching_attempts` is more descriptive (and would have made this more obvious).

Reply:
I don't think the naming is a problem, but rather that returning a list here was just a bad idea on my part :P. At the time it wasn't used and I wasn't sure what to return here. I'm gonna add it to the unit tests now.